### PR TITLE
WIP: POC Feature/renderer

### PR DESCRIPTION
**Pull request by lbb**

### CODE REVIEW

File: api/pkg/controller/resources/load_files.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ const (
 )
 
 // LoadDeploymentFile loads a k8s yaml deployment from disk and returns a Deployment struct
+// TODO(realfake): This function calls net.LookupHost, move it out of the function!
```
Comment (mrIncompetent) [inline api/pkg/controller/resources/load_files.go:25]: Maybe a new variable in the cluster + we extend the existing step which fills out the other cluster address fields?

Comment (lbb) [reply api/pkg/controller/resources/load_files.go:25]: Yeah, wanted to do this anyways, but at the end. (In another PR)

**Comment by mrIncompetent** [comment]
I assume this can be closed?

**Comment by lbb** [comment]
Yes it can be closed it is deprecated, regarding the latests discussions.
/close

**Pull request closed without merging**